app/src/main/java/org/mozilla/fenix/search/toolbar/ToolbarView.kt

```diff
@@ -135,7 +136,12 @@ class ToolbarView(
             // we have the most up to date text
             interactor.onTextChanged(view.url.toString())
 
-            view.editMode()
+            // If search terms are displayed, move the cursor at the end instead of selecting all text.
+            if (settings.showUnifiedSearchFeature && searchState.searchTerms.isNotBlank()) {
+                view.editMode(Toolbar.CursorPlacement.END)
```

Comment:
```suggestion
                view.editMode(cursorPlacement = Toolbar.CursorPlacement.END)
```
Let's add the named parameter here since this will make it even easier to understand.